examples/widgets/bottom_sheet.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
I'll leave the demo out of the commit.  I found it useful for working out the feature but it doesn't need to part of the repository.